recipes/darknet/all/patches/include_conan_flags.patch

```diff
@@ -0,0 +1,15 @@
+diff -ru a/Makefile b/Makefile
+--- a/Makefile
++++ b/Makefile
+@@ -25,9 +25,9 @@
```

Comment:
I think you will also need to patch CC and CPP, because it is hardcoded to gcc.
Something like CC?=gcc and CPP?=g++ would be ok.